### PR TITLE
Mask RCNN DApp

### PR DESCRIPTION
**Pull request by trancept**
Semantic segmentation using Mask-RCNN implementation from Matternet with Keras and Tensorflow. Just send an image URL and get back the segmentation. Feel free to ask for improvement.

### CODE REVIEW

**Comment by trancept** [comment]
Hello Ugo,
The image have been updated to:
- Need an image URL as parameter. I've removed the fallback to sample pictures.
- Produce the consensus.iexec with deterministic behavior
- Reduce his size from 3.41GB to 2.25GB. There is still room for improvement as the size at execution time is only 1.7GB.

**Comment by trancept** [comment]
New PR created : https://github.com/iExecBlockchainComputing/iexec-dapps-registry/pull/79
